SPECS-EXTENDED/catatonit/catatonit.spec

```diff
@@ -13,7 +13,7 @@ BuildRequires: automake
 BuildRequires: file
 BuildRequires: gcc
 BuildRequires: git
-BuildRequires: glibc-static >= 2.35-6%{?dist}
+BuildRequires: glibc-static >= 2.38-1%{?dist}
```

Comment:
ditto

Reply:
Yes, we need to ensure that these packages are rebuilt/re-released with the latest glibc static components, and the qualifier is needed for that. More info in https://github.com/microsoft/CBL-Mariner/pull/3748